### PR TITLE
feat!: Upgrade Powershell Dict

### PR DESCRIPTION
**Pull request by KieranRobson**
# Add/Fix Dictionary

Dictionary: Powershell

## Description

Upgrade as part of #705 

## References

NA 

## Checklist

- [x] By submitting this issue, you agree to follow our
      [Code of Conduct](https://github.com/streetsidesoftware/cspell-dicts/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by KieranRobson** [comment]
Will fix the lint when I'm back at my laptop! Woops

**Comment by KieranRobson** [comment]
```
196/262 .\dictionaries\powershell\dict\powershell.txt cached X
d:\Projects\Misc Project\cspell-dicts\dictionaries\powershell\dict\powershell.txt:254:43 - Unknown word (Unpublish)
d:\Projects\Misc Project\cspell-dicts\dictionaries\powershell\dict\powershell.txt:291:47 - Unknown word (Unprotect)
```

Linting cant be fixed due to the above

File: dictionaries/powershell/package.json [hunk 1]
```diff
@@ -11,8 +11,8 @@
     "./cspell-ext.json": "./cspell-ext.json"
   },
   "scripts": {
-    "build": "cspell-tools compile \"powershell.txt\" -o .",
-    "test": "head -n 100 \"powershell.txt\" | cspell -v -c ./cspell-ext.json \"--local=*\" \"--languageId=powershell\" stdin",
+    "build": "cspell-tools-cli compile --keep-raw-case src/powershell.txt --no-compress -o ./dict",
```
Comment (Jason3S) [inline dictionaries/powershell/package.json:14]: I think `--use-legacy-splitter` is needed.

Comment (KieranRobson) [reply dictionaries/powershell/package.json]: Done!

**Pull request merged**